scalableminds/webknossos
# docker: dev service

* adding docker-compose dev service (with persistent mongo)
* cleanup

Steps to test:
- run docker-compose up dev
- open localhost:9000
- change things. should be consistent across restarts

------


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1663/create?referer=github" target="_blank">Log Time</a>
